model/task_lifecycle.go

```diff
@@ -592,17 +592,29 @@ func MarkEnd(t *task.Task, caller string, finishTime time.Time, detail *apimodel
 // UpdateBlockedDependencies traverses the dependency graph and recursively sets each
 // parent dependency as unattainable in depending tasks.
 func UpdateBlockedDependencies(t *task.Task) error {
+	const maxStackTraceSize = 5
 	dependentTasks, err := t.FindAllUnmarkedBlockedDependencies()
 	if err != nil {
 		return errors.Wrapf(err, "can't get tasks depending on task '%s'", t.Id)
 	}
 
 	if t.IsPartOfSingleHostTaskGroup() && len(dependentTasks) > 0 {
+		depTaskIds := []string{}
+		for _, depTask := range dependentTasks {
+			depTaskIds = append(depTaskIds, depTask.Id)
+		}
+		stack := message.NewStack(2, "").Raw().(message.StackTrace).Frames
```

Comment:
Jonathan wrote this but it seems correct to skip UpdateBlockedDependencies since that's where this is being called from; we specifically want to see what functions are calling it.